packages/contracts/contracts/Semaphore.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.4;
+pragma solidity 0.8.19;
```

Comment:
Why not use `0.8.23` instead of `0.8.19`?

Reply:
It's the one Hardhat sets when you init a project. I think it's better to use it for compatibility. There are some warnings with the latest.

Reply:
Which warnings? I tested with `0.8.23` and I didn't see any warnings.